internal/tests/pytests/util/run_util/test_run_util.py

```diff
@@ -2,45 +2,81 @@
-                        'INPUT_BASE', 'OUTPUT_BASE', 'METPLUS_CONF',
-                        'TMP_DIR', 'STAGING_DIR', 'FILE_LISTS_DIR', 'CONVERT',
-                        'GEMPAKTOCF_JAR', 'GFDL_TRACKER_EXEC', 
-                        'PROCESS_LIST', 'OMP_NUM_THREADS',
-                        'SCRUB_STAGING_DIR', 'LOG_METPLUS', 'LOG_DIR',
-                        'LOG_TIMESTAMP_TEMPLATE', 'LOG_TIMESTAMP_USE_DATATIME',
-                        'LOG_MET_OUTPUT_TO_METPLUS', 'LOG_LEVEL',
-                        'LOG_LEVEL_TERMINAL', 'LOG_MET_VERBOSITY',
-                        'LOG_LINE_FORMAT', 'LOG_ERR_LINE_FORMAT',
-                        'LOG_DEBUG_LINE_FORMAT', 'LOG_INFO_LINE_FORMAT',
-                        'LOG_LINE_DATE_FORMAT', 'DO_NOT_RUN_EXE', 'CONFIG_INPUT',
-                        'RUN_ID', 'LOG_TIMESTAMP', 'METPLUS_BASE', 'PARM_BASE',
-                        'METPLUS_VERSION']
+
+EXPECTED_CONFIG_KEYS = [
+    'CLOCK_TIME',
+    'MET_INSTALL_DIR',
+    'MET_BIN_DIR',
+    'INPUT_BASE',
+    'OUTPUT_BASE',
+    'METPLUS_CONF',
+    'TMP_DIR',
+    'STAGING_DIR',
+    'FILE_LISTS_DIR',
+    'CONVERT',
+    'GEMPAKTOCF_JAR',
+    'GFDL_TRACKER_EXEC',
+    'PROCESS_LIST',
+    'OMP_NUM_THREADS',
+    'SCRUB_STAGING_DIR',
+    'LOG_METPLUS',
+    'LOG_DIR',
+    'LOG_TIMESTAMP_TEMPLATE',
+    'LOG_TIMESTAMP_USE_DATATIME',
+    'LOG_MET_OUTPUT_TO_METPLUS',
+    'LOG_LEVEL',
+    'LOG_LEVEL_TERMINAL',
+    'LOG_MET_VERBOSITY',
+    'LOG_LINE_FORMAT',
+    'LOG_ERR_LINE_FORMAT',
+    'LOG_DEBUG_LINE_FORMAT',
+    'LOG_INFO_LINE_FORMAT',
+    'LOG_LINE_DATE_FORMAT',
+    'DO_NOT_RUN_EXE',
+    'CONFIG_INPUT',
+    'RUN_ID',
+    'LOG_TIMESTAMP',
+    'METPLUS_BASE',
+    'PARM_BASE',
+    'METPLUS_VERSION',
+]
 
 
 def get_run_util_configs(conf_name):
     script_dir = os.path.dirname(__file__)
     return [os.path.join(script_dir, conf_name)]
 
 
+def get_config_from_file(conf_file='run_util.conf'):
+    """
+    Make a config object from .conf file.
+    Similar to fixture 'metplus_config', but calling that fixture
+    interfers with how output is written. Use this instead when
+    invoking pytest fixture 'capfd'.
+    """
+    conf_inputs = get_run_util_configs(conf_file)
+    return ru.pre_run_setup(conf_inputs)
+
+
 @pytest.mark.util
 def test_pre_run_setup():
-    conf_inputs = get_run_util_configs('run_util.conf')
-    actual = ru.pre_run_setup(conf_inputs)
-    
+    actual = get_config_from_file()
+
     # check all config keys are set correctly
     assert sorted(actual.keys('config')) == sorted(EXPECTED_CONFIG_KEYS)
-    
+
     # spot check a few specific items
     expected_stage = os.path.join(actual.get('config', 'OUTPUT_BASE'), 'stage')
     assert actual.get('config', 'STAGING_DIR') == expected_stage
-    assert actual.get('user_env_vars', 'GODS_OF_WEATHER', 'Indra_Thor_Zeus') 
-    
-    
+    assert actual.get('user_env_vars', 'GODS_OF_WEATHER', 'Indra_Thor_Zeus')
```

Comment:
I think this assert always passes because a default value is specified. Should this be:
```suggestion
    assert actual.get('user_env_vars', 'GODS_OF_WEATHER') == 'Indra_Thor_Zeus'
```

Reply:
Good pick up, thanks. Fixed in latest commit.